cypress/integration/auth.ts

```diff
@@ -7,21 +7,16 @@ describe("Auth", () => {
     cy.url().should("include", "/login");
   });
 
-  it("Redirects user back to the route they were trying to visit after login", () => {
+  it("Redirects user to My Patches page after logging in.", () => {
```

Comment:
Why did this change? I think we should still preserve the previous route after a successful login